City-of-Helsinki/hel-fi-drupal-grants
# feat: AU-2407: Changes to the "Resend applications" form

# [AU-2407](https://helsinkisolutionoffice.atlassian.net/browse/AU-2407)

## What was done

This pull request implements a few changes to the "[Resend applications](https://hel-fi-drupal-grant-applications.docker.so/fi/admin/tools/resend-applications)" form:

1. Implemented row colors based on message sender.
2. Prettified the JSON output.
3. Removed the "Avus2 has received the message" column.
4. Removed the "Resend this message" checkbox from any messages sent by Avus2.
5. Fixed a typo.

## How to install

Make sure your instance is up and running on correct branch.
* `git checkout feature/AU-2407-atv-tools-fix`
* `make fresh`
* `make drush-cr`

## How to test
- Go to the "[Resend applications](https://hel-fi-drupal-grant-applications.docker.so/fi/admin/tools/resend-applications)" form and test it with the ID `LOCALT-067-0000009`. Make sure your page looks something like this:

![Screenshot 2024-05-02 at 13-39-02 Admin Resend applications Avustusasiointi](https://github.com/City-of-Helsinki/hel-fi-drupal-grants/assets/117807015/51f222f9-4d2e-4827-a26c-240b3b8714e3)



[AU-2407]: https://helsinkisolutionoffice.atlassian.net/browse/AU-2407?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
